### PR TITLE
Remove paragraph about .htaccess files

### PR DESCRIPTION
**Pull request by l-vo**
Htaccess files aren't in SF repository anymore (since sf4 and Flex, it's in the apache-pack optional recipe)

### CODE REVIEW

File: setup/web_server_configuration.rst [hunk 1]
```diff
@@ -374,13 +374,6 @@ The **minimum configuration** to get your application running under Nginx is:
     After you deploy to production, make sure that you **cannot** access the ``index.php``
     script (i.e. ``http://example.com/index.php``).
 
-.. note::
-
-    By default, Symfony applications include several ``.htaccess`` files to
```
Comment (noniagriconomie) [inline setup/web_server_configuration.rst:379]: Why not keeping it but instead document that it requires the optional package?

Comment (l-vo) [reply setup/web_server_configuration.rst:379]: Because it's already documented in the beginning of the page.

**Review by wouterj — APPROVED** [review]

**Review by xabbuh — APPROVED** [review]

**Comment by wouterj** [comment]
Hi Laurent! Thanks for yet another great fix :)

**Comment by l-vo** [comment]
@wouterj You're welcome :)

**Pull request merged**